themes/src/main/resources/theme/keycloak.v2/account/resources/public/layout.css

```diff
@@ -23,3 +23,38 @@
   --pf-c-content--heading--FontFamily: "Red Hat Text", Overpass, overpass, helvetica, arial, sans-serif;
 }
 /* End TODO */
+
+/* Device Activity screen */
+.signed-in-device-list .pf-c-data-list__item-row {
+  --pf-c-data-list__item-row--PaddingRight: 0;
+  --pf-c-data-list__item-row--PaddingLeft: 0;
+}
+
+.signed-in-device-list .pf-c-data-list__expandable-content-body {
+  --pf-c-data-list__expandable-content-body--PaddingRight: 0;
+}
+
+.signed-in-device-grid {
+  grid-template-columns: auto repeat(11, [col-start] 1fr);
+}
+
+.signed-in-device-list.pf-c-data-list {
+  --pf-c-data-list--sm--BorderTopWidth: 0;
+}
+
+@media (max-width: 576px) {
```

Comment:
It's best to always use min-width media queries. Set your styles for mobile, then use min-width to adjust. Mixing can cause problems right at the breakpoints.